apps/verify/behaviours/get-cookies.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+const cookie = require('../models/cookie');
+const generatecookie = async() => {
+  // requires getting an email from the magiclink
+  return await cookie.writeCookie({'user-email': 'bat@mail.com'});
```

Comment:
Don't use real email addresses -- use something [reserved](https://tools.ietf.org/html/rfc2606).
```suggestion
  return await cookie.writeCookie({'user-email': 'bat@example.com'});
```